### PR TITLE
Enable tolerations support in elasticsearch config

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

### CODE REVIEW

**Comment by kevinearls** [comment]
@objectiser @jpkrohling @rubenvp8510 Please review

File: pkg/storage/elasticsearch/v1/types.go [hunk 1]
```diff
@@ -104,7 +104,8 @@ const (
 type ElasticsearchNodeSpec struct {
 	Image        string                  `json:"image,omitempty"`
 	Resources    v1.ResourceRequirements `json:"resources"`
-	NodeSelector map[string]string       `json:"nodeSelector,omitempty"`
+	NodeSelector map[string]string       `json:"nodeSelector"`
```
Comment (objectiser) [inline pkg/storage/elasticsearch/v1/types.go:107]: Should this still be `json:"nodeSelector,omitempty"`?

**Review by objectiser — COMMENTED** [review]
Is there any way to add tests for this? Has it been tested manually?

**Review by jpkrohling — APPROVED** [review]
LGTM, we might need to mention this in the docs or in an example.

**Pull request merged**